.github/workflows/sonarcloud.yml

```diff
@@ -2,9 +2,9 @@ name: SonarCloud
 
 on:
   push:
-    branches: [ NetStandard ]
+    branches: [ master ]
```

Comment:
If NetStandard branch is going to stay, we add it also after comma.

Reply:
Whoops!  Good catch.  I forgot about that.